packages/field-base/src/field-mixin.js

```diff
@@ -113,14 +131,22 @@ export const FieldMixin = (superclass) =>
       }
     }
 
+    _accessibleNameChanged(accessibleName) {
```

Comment:
Let's add `@private` JSDoc annotations here and below.

Reply:
Added.